rules/RULE_5_3_A_provide_doxygen_function_comment_on_function_in_header.py

```diff
@@ -44,13 +44,13 @@ def RunRule(lexer, fullName, decl, contextStack, context) :
     if ext == ".h" :
         upperBlock = contextStack.SigPeek()
         # modification to make doxygen compulsory for even private functions
-        if upperBlock is None:  # != None: and upperBlock.type == "CLASS_BLOCK" and upperBlock.additional == "PRIVATE":
+        if upperBlock is not None and upperBlock.type == "CLASS_BLOCK" and upperBlock.additional == "PRIVATE":

```

Comment:
Won't this result in failure in the test case where the function is private? (I'm unable to run tests on my mobile :smile: )